### DISCUSSION TITLE
Installation Problem: Repository + Release File

### DISCUSSION THREAD

**Discussion by bbillp** [aaronwmorris/indi-allsky#1682] [General]
11/27/2024 
Ubuntu 22 on a Lenovo Laptop
I am unable to get past these messages during installation
The repository does not have a release file
updating from this repository can’t be done securely

Thanks in advance

![image](https://github.com/user-attachments/assets/e37a6996-b4f7-4fef-9947-5d8fac0ba52d)

**Comment by aaronwmorris** [comment]
It looks like you setup a PPA for ok2cqr that is not valid for some reason.  If you can look under `/etc/apt/sources.list.d/` and delete the repo file related to this PPA, you should be able to get past this.

**Comment by bbillp** [comment]
OK I’ll do it but the entire download was performed 11/27/2024 from this github site.  Thanks for the help.

**Reply by aaronwmorris** [reply]
The ok2cqr PPA repository is something that was not setup by indi-allsky.  That repository existed before indi-allsky was installed.

**Comment by bbillp** [comment]
changed the name of the ok2cqr file to backup and then loaded this missing LIB and it worked !! Very very nice software.   Thanks   
![image](https://github.com/user-attachments/assets/052ba455-e164-4ad9-b0b8-15afb21e55d4)

**Reply by aaronwmorris** [reply]
Excellent!
